### ISSUE TITLE
issue with package.json

### ISSUE THREAD

**Issue by nate2009** [Microsoft/vscode#9171]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1. Cannot read file /home/iga/.vscode/extensions/ms-vscode.cpptools-0.7.1/package.json: ENOENT: no such file or directory, open '/home/iga/.vscode/extensions/ms-vscode.cpptools-0.7.1/package.json'.
2.

**Comment by weinand** [comment]
@nate2009 in what situation does this occur? Please provide **steps** not just an error message.

**Comment by weinand** [comment]
This issue was moved to Microsoft/vscppsamples#99

**Comment by nate2009** [comment]
It was a mistake I made after updating it I was using the quick start to open it. I had to delete the programs quick start and go directly to the folder and start the program to get the updated program to run. I don't know why it didn't run the updated version but I had to delete the old version of the program on the quick start and stick the new updated version to get it to run with the update. 

Sent from Yahoo Mail on Android 
 
  On Wed, Jul 13, 2016 at 3:31 PM, Andre Weinand<notifications@github.com> wrote:   
Closed #9171.
